code/modules/jobs/job_types/station_engineer.dm

```diff
@@ -47,18 +47,22 @@
 	backpack_contents = list(/obj/item/modular_computer/tablet/preset/advanced=1)
 
 /datum/outfit/job/engineer/gloved
-	name = "Station Engineer (Gloves)"
+	name = "Mechanic (Gloves)"
 	gloves = /obj/item/clothing/gloves/color/yellow
 
 /datum/outfit/job/engineer/gloved/rig
-	name = "Station Engineer (Hardsuit)"
+	name = "Mechanic (Hardsuit)"
```

Comment:
Since the hardsuit is Nanotrasen equipment you might want to either change this to use the indie engineering soft suit, or leave this one with the Nanotrasen outfit underneath and make it a subtype of the NT Engineer outfit.

Reply:
changed to indie space suit